source/grid/grid_generator.cc

```diff
@@ -3919,6 +3899,16 @@ namespace GridGenerator
     std::vector<Point<3>>    vertices;
     std::vector<CellData<3>> cells;
 
+    // Corner points of the cube [-1,1]^3
+    static const std::array<Point<3>, 8> hexahedron = {{{-1, -1, -1}, //
+                                                        {+1, -1, -1}, //
+                                                        {-1, +1, -1}, //
+                                                        {+1, +1, -1}, //
+                                                        {-1, -1, +1}, //
+                                                        {+1, -1, +1}, //
+                                                        {-1, +1, +1}, //
+                                                        {+1, +1, +1}}};
```

Comment:
Would it be possible to mark this as `constexpr`, or would that require C++14?

Reply:
I couldn't find a (suitably simple) way to make the `Point` constructor used here `constexpr` using just C++11. That requires to make `Tensor` `constexpr` which is not really easy due to all our customization for specific `Number` template parameters.

Reply:
That's right; I forgot about that.